js/app/src/components/Audio/Audio.svelte

```diff
@@ -41,6 +44,13 @@
 	let _value: null | FileData;
 	$: _value = normalise_file(value, root, root_url);
 
+	$: {
```

Comment:
I think we're trying to move away from JSON.stringify. I think the `Label` component has the "preferred" way but gonna defer to @pngwn

Reply:
Ah good to know! Will take a look later